app/components/CoinListCell/CoinListCell.js

```diff
@@ -1,9 +1,13 @@
-// @flow
 import React, { Component } from 'react'
 import { View, Text, Image, LayoutAnimation, TouchableOpacity } from 'react-native'
+import PropTypes from 'prop-types'
 import coinListCellStyles from './CoinListCellStyles'
 import CoinListCellRangeView from './CoinListCellRangeView/CoinListCellRangeView'
-import CoinListCellHeader from './CoinListCellHeader/CoinListCellHeader'
+import CoinListCellFavoritesButton from './CoinListCellFavoritesButton/CoinListCellFavoritesButton'
```

Comment:
Did you want to utilize the `index` pattern here? Something like ` './CoinListCellFavoritesButton/index'`?
It just seems weird to have `CoinListCellFavoritesButton` x 2

Reply:
Yup! Lets do that.

Reply:
Wait where do you see it used twice?

Reply:
Oh I see what you mean. Right right.